--- a/qb/qb.params.sh
+++ b/qb/qb.params.sh
@@ -62,7 +62,7 @@ opt_exists() # $opt is returned if exist
 	err="$2"
 	eval "set -- $OPTS"
 	for OPT do [ "$opt" = "$OPT" ] && return; done
-	die 1 "Unknown option $err"
+	echo "Unknown option $err"
 }
 
 parse_input() # Parse stuff :V
@@ -106,7 +106,7 @@ parse_input() # Parse stuff :V
 			-h|--help) print_help; exit 0;;
 			--) break ;;
 			'') : ;;
-			*) die 1 "Unknown option $1";;
+			*) echo "Unknown option $1";;
 		esac
 		shift
 	done
